docs/developer-docs/latest/setup-deployment-guides/deployment/hosting-guides/heroku.md

```diff
@@ -244,14 +244,16 @@ module.exports = ({ env }) => ({
   })
 ```
 
-You will also need to set the environment variable in Heroku for the `MY_HEROKU_URL` and `APP_KEYS`. This will populate the variables with something like `https://your-app.herokuapp.com/` and `dsfhasbvvfwfcerterzer+n1w==,afjdsagfsauzuwzref6==,kjdbgjerhgh6wireg==,jkssdhgjaksdgkjbsdg==` respectively.
+You will also need to set the environment variable in Heroku for the `MY_HEROKU_URL`, `APP_KEYS`, `API_TOKEN_SALT`, `ADMIN_JWT_SECRET`, and `JWT_SECRET`. This will populate the variables with something like `https://your-app.herokuapp.com/` and various random keys from the `.env` file locally. In some cases it is recommended to create new random secrets instead and there are various methods to do so.
```

Comment:
Should we add documentation (not here) about creating secrets? It seems to come up regularly.